### DISCUSSION TITLE
Output of Nmap-Scan missing

### DISCUSSION THREAD

**Discussion by p0w1** [projectdiscovery/naabu#591] [Q&A] [answered]
If naabu is started with "-nmap-cli" the output of Nmap is not stored in the output file:
`naabu -host 127.0.0.1 -o test -nmap-cli 'nmap -sV'`

### Please describe your feature request:
The output of Nmap should be stored in a separate output file

**Comment by ehsandeep** [comment, marked as answer]
@p0w1, you need to write output cmd within nmap cmd to write nmap output to a file.

For example

```
naabu -host 127.0.0.1 -o test -nmap-cli 'nmap -sV -oX test'
```

**Reply by p0w1** [reply]
True, thanks for the quick feedback. A more elegant solution for the future would be, if the Nmap-Output is inserted in the Naabu-JSON output.
